src/renderer/components/ChildWindow/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useMemo } from 'react';
+import { createPortal } from 'react-dom';
+import { useAppFeature } from 'renderer/contexts/AppFeatureProvider';
+
+type Props = {
+  onClosed?: () => void;
+  children: React.ReactNode;
+  options?: string;
+};
+
+export const ChildWindowContext = React.createContext<Window | null>(null);
+
+export function ChildWindow({ onClosed, children, options }: Props) {
```

Comment:
We can add one more property `{ onClosed, children, options, title }`. We need title to set the title